src/setuptools_scm/__init__.py

```diff
@@ -149,6 +149,7 @@ def _get_version(config: Configuration) -> str | None:
         parsed_version,
         version_scheme=config.version_scheme,
         local_scheme=config.local_scheme,
+        absolute_root=config.absolute_root,
```

Comment:
I have a larger refactoring in the pipeline that should enable not needing to pass that

I'll elaborate once I get the time

Reply:
thank you so much!